File: 151002/Kuvshinov/lab2/DTO/ResponseDTO/NoteResponseDto.cs
using DC.Models;

namespace DC.DTO.ResponseDTO
{
	public class NoteResponseDto
	{
		public long Id { get; set; }

		public long StoryId { get; set; }

		public Story Story { get; set; }

		public string Content { get; set; }
	}
}

File: 151003/Haritonenko/task2/src/main/java/by/haritonenko/jpa/dto/response/NoteResponseDto.java
package by.haritonenko.jpa.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class NoteResponseDto {

    private Long id;
    private Long storyId;
    private String content;
}
